assets/js/modules/analytics-4/components/dashboard/DashboardAllTrafficWidgetGA4/UserDimensionsPieChart.js

```diff
@@ -171,9 +171,10 @@ export default function UserDimensionsPieChart( props ) {
 		maxSlices: 5,
 		withOthers: true,
 		tooltipCallback: ( row, previousDateRangeRow, rowData ) => {
+			const rowValue = row?.metricValues?.[ 0 ]?.value || 0;
 			const difference =
-				previousDateRangeRow.metricValues?.[ 0 ].value > 0
-					? ( row.metricValues?.[ 0 ].value * 100 ) /
+				previousDateRangeRow?.metricValues?.[ 0 ].value > 0
+					? ( rowValue * 100 ) /
 							previousDateRangeRow.metricValues?.[ 0 ].value -
```

Comment:
Same here, we need to use `?.value`.

Reply:
@eugene-manuilov this is not required as it is inside the conditional check of `previousDateRangeRow?.metricValues?.[ 0 ]?.value > 0` but will update for consistency.

Reply:
This is required: 
![image](https://github.com/user-attachments/assets/4e2228b8-b6f5-4e4a-95bc-44ae1c85ca58)